Partition_2/doc/Partition_2/Concepts/YMonotonePartitionTraits_2.h

```diff
@@ -1,89 +0,0 @@
-/*!
```

Comment:
Bump on comment: https://github.com/CGAL/cgal/pull/3848#pullrequestreview-225973792.

Also, is there really no need for `YMonotonePartitionTraits_2` but still a need for `YMonotonePartitionIsValidTraits_2`?

Reply:
It still 
has a specific functor.